### PR TITLE
feat(account-lib): add utils to validate tx and block hash

### PR DESCRIPTION
**Pull request by nachofm**
STLX-10625

### CODE REVIEW

File: modules/account-lib/test/unit/coin/dot/utils.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+describe('utils', () => {
+  it('should validate addresses correctly', () => {
+    should.equal(utils.isValidAddress(accounts.account1.address), true);
+    should.equal(utils.isValidAddress(accounts.account2.address), true);
+    should.equal(utils.isValidAddress(accounts.account3.address), true);
+    should.equal(utils.isValidAddress(accounts.account4.address), true);
+  });
+
+  it('should validate block hash correctly', () => {
+    should.equal(utils.isValidBlockId(blockHash.block.hash), true);
```
Comment (argjv) [inline modules/account-lib/test/unit/coin/dot/utils.ts:14]: Can you add a couple more block hashes? Also, need error cases tests

File: modules/account-lib/test/unit/coin/dot/utils.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import should from 'should';
+import utils from '../../../../src/coin/dot/utils';
+import { accounts, blockHash } from '../../../resources/dot';
+
+describe('utils', () => {
```
Comment (argjv) [inline modules/account-lib/test/unit/coin/dot/utils.ts:5]: Please add a few error scenarios

File: modules/account-lib/src/coin/dot/utils.ts [hunk 3]
```diff
@@ -52,7 +51,8 @@ export class Utils implements BaseUtils {
 
   /** @inheritdoc */
   isValidSignature(signature: string): boolean {
-    throw new NotImplementedError('method not implemented');
+    const signatureU8a = u8aToU8a(signature);
+    return [64, 65, 66].includes(signatureU8a.length);
```
Comment (argjv) [inline modules/account-lib/src/coin/dot/utils.ts:55]: This is pretty much what `signatureVerify()` does in the polkadot lib

Comment (raovinnakota) [reply modules/account-lib/src/coin/dot/utils.ts:55]: Yeah - does it make sense to just have it use it like this:
`const { isValid } = signatureVerify(message, signature, alice.address);`

source: https://polkadot.js.org/docs/keyring/start/sign-verify/

Comment (argjv) [reply modules/account-lib/src/coin/dot/utils.ts:55]: Oh, we have that method right below here `verifySignature(signedMessage: string, signature: string, address: string): boolean`

**Review by raovinnakota — APPROVED** [review]
LGTM - verifySignature nit addressed

**Pull request merged**